trillian/ctfe/handlers.go

```diff
@@ -875,7 +875,7 @@ func parseGetSTHConsistencyRange(r *http.Request) (int64, int64, error) {
 	if first < 0 || second < 0 {
 		return 0, 0, fmt.Errorf("first and second params cannot be <0: %d %d", first, second)
 	}
-	if second <= first {
+	if second < first {
 		return 0, 0, fmt.Errorf("invalid first, second params: %d %d", first, second)
 	}
 
```

Comment:
Did you consider just returning an empty proof in here and short-circuiting the RPC?

Reply:
I did but we then can't validate that the parameters are inside the valid tree size range.

Reply:
Though actually doesn't it cache STHs now? Perhaps we can.